data/json/items/melee.json

```diff
@@ -2092,6 +2092,40 @@
         "qualities": [["BUTCHER", -50]]
     },
     {
+        "type":"GENERIC",
```

Comment:
Adding two items just to make butcher value of claws jsonable is an overkill.
Just hardcode their values for now, later on you can move them to mutation jsons themselves, without a dummy item.